pages/install/conda-install.rst

```diff
@@ -121,3 +140,22 @@ remove and then reinstall Cantera:
 
 Alternatively, you can remove the ``ct-dev`` environment and follow Options 1 or 2
 above to create a new environment.
+
+.. _sec-conda-matlab-interface:
+
+MATLAB interface
+================
+
+Cantera's MATLAB interface can be installed from the ``cantera`` channel. In this
+example, the command will create a new conda environment named ``ct-env``. From the
+command line (or the Anaconda Prompt on Windows), run:
+
+.. code:: shell
+
+   conda create --name ct-env --channel cantera cantera cantera-matlab
+
+This will create an environment with Cantera's Python and MATLAB interfaces. Even if you
+plan to use Cantera via MATLAB, the Python interface is required to convert input files.
+In this case, Cantera must be installed from the ``cantera`` channel, so we add
+``--channel cantera`` and to tell Conda to look at the ``cantera`` channel in addition
+to the default channels.
```

Comment:
Do people have to start MATLAB out of the conda environment? Are there any adjustments needed for matlabpath?

Reply:
The intention is that this is handled by a post-install script in the package which should be executed automatically. I'm not totally certain about whether or not post-install scripts are still executed by Conda or whether they've disabled that for security reasons.

Reply:
Thank you for the clarification. As the expected behavior isn't explicit (i.e. post-install scripts handle that everything is set up correctly), I was hoping that clarifying text could be added here.